libraries/helpers.rb

```diff
@@ -31,5 +31,39 @@ def get_source_home_dir(new_resource, node)
     def get_source_root_dir(new_resource, node)
       new_resource.dir || node.ark[:prefix_root]
     end
+
+    # This method takes a hash, but will convert to mash
```

Comment:
I'd like to have a "regular" RDoc/[YARD](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md) documentation for these Ruby methods, including at least one `@example` section. Do you think it's a good idea?